docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3.6'  
+services:  
+  analyzer:
+    image: codechecker
+    command: tail -F /dev/null
```

Comment:
why is this needed?

Reply:
This is not needed, I have just forget to remove this.